scm-ui/ui-api/src/repositories.ts

```diff
@@ -153,103 +151,103 @@ export const useDeleteRepository = (options?: UseDeleteRepositoryOptions) => {
         if (options?.onSuccess) {
           options.onSuccess(repository);
         }
-        await queryClient.removeQueries(repoQueryKey(repository));
+        queryClient.removeQueries(repoQueryKey(repository));
```

Comment:
Do we no longer have to wait for the removal?

Reply:
No. `queryClient.removeQueries` does not return a `Promise`.